server/index.js

```diff
@@ -86,7 +86,12 @@ db.sequelize
     console.error("Unable to connect to the database:", err);
   });
 
-const server = app.listen(port, () => {
-  console.log(`Listening on port ${server.address().port}`);
-});
+(async () => {
+  await db.sequelize.sync({ force: true });
```

Comment:
the `force: true` will drop the table every time we start the server, is there a reason why we want to do that?

Reply:
Thanks for pointing it out. I forgot to take that out while debugging.